index.bs

```diff
@@ -12182,9 +12450,25 @@ asynchronous sections can't be reordered.
 <h3 id="rendering-loop">
 Rendering an Audio Graph</h3>
 
+<div class="correction proposed" id="c2450-10">
+  <span class="marker">Proposed addition
+    <a href="https://github.com/WebAudio/web-audio-api/issues/2450">Issue 2450</a>-10.
+  </span>
+  Allow specifying a different render quantum size
+  <div class="amendment-buttons"></div>
+  <del>
 Rendering an audio graph is done in blocks of 128 samples-frames. A
-block of 128 samples-frames is called a <dfn>render quantum</dfn>, and
-the <dfn>render quantum size</dfn> is 128.
+block of 128 samples-frames is called a <a>render quantum</a>, and
+the <a>render quantum size</a> is 128.
+  </del>
+  <ins>
+Rendering an audio graph is done in blocks of samples-frames, the size of this
```

Comment:
To make it read better:

Audio graph rendering is done in blocks of sample-frames, with the size of each block remaining constant for the lifetime of a BaseAudioContext. The number of sample-frames in a block is called `render quantum size`, and the block itself is called a `render quantum`. Its default value is 128, and it can be configured by setting AudioContextOptions/renderSizeHint.